src/disable-amp.js

```diff
@@ -40,16 +38,20 @@ function preventAmp() {
         // eslint-disable-next-line no-param-reassign
         el[expando] = true;
 
+        const url = el.getAttribute('data-amp-cur');
+        el.setAttribute('href', url);
+
         el.addEventListener('click', (e) => {
             e.preventDefault();
             e.stopPropagation();
-            const url = el.getAttribute('data-amp-cur');
             // https://github.com/AdguardTeam/DisableAMP/pull/15
             document.location.href = url;
         }, true);
         hideAmpIcon(el);
     });
 }
 
-preventAmp();
-observeDomChanges(preventAmp);
+if (document.URL.includes("google")) {
```

Comment:
it's too wide
maybe something like `document.location.origin.endsWith("google.com")`

Reply:
But endsWith don't work only if entire URL end with google.com? This could generate an error.

Reply:
Ok... I thought location.origin return entire link (thanks for suggestion), however I don't think it's possible to use "google.com" as a search key in the condition, so unless further suggestions this is my fix: https://github.com/AdguardTeam/DisableAMP/pull/19/commits/1b8c21f59a441e8d075b620faf898fc9277755d5